spyder/widgets/onecolumntree.py

```diff
@@ -62,15 +62,14 @@ def setup_common_actions(self):
                                      icon=ima.icon('restore'),
                                      triggered=self.restore)
         self.collapse_selection_action = create_action(self,
-                                     text=_('Collapse selection'),
+                                     text=_('Collapse section'),
                                      icon=ima.icon('collapse_selection'),
                                      triggered=self.collapse_selection)
         self.expand_selection_action = create_action(self,
-                                     text=_('Expand selection'),
+                                     text=_('Expand section'),

```

Comment:
Please do the same as suggested above. Thanks

Reply:
@goanpeca would you like me to change also the other 3 actions that are above these two to match the same format?

Reply:
That would be nice but I do not think is necessary.